### ISSUE TITLE
Implement Code Suite for Fraud Modeling

### ISSUE THREAD

**Issue by advaithsrao** [advaithsrao/Fraud-Detector#4]
Go through prior art of ethical techniques around text classification. What can we find and use?
Write optimal code -> preferably a structured data pipeline that does the following,

- [ ] Pulls data based on label data logic & runs preprocessing pipeline (Spark?) #1 
- [ ] Pretrained RoBERTa training code from #2 
- [ ] Testing module from #3 

The above should be done using one python script call for a user.

**Note:** We will also integrate the ethical codes using ticket #8
